### ISSUE TITLE
2.10.5 break scripts for www.sis001.com

### ISSUE THREAD

**Issue by 15556133244** [violentmonkey/violentmonkey#581]
```
// ==UserScript==
// @name New Script
// @namespace Violentmonkey Scripts
// @match https://www.sis001.com/*
// @grant none
// ==/UserScript==
console.log('test')
```
log nothing
SyntaxError: unexpected token: ')'
injected.js:1

refresh may fix it, strange.

**Comment by gera2ld** [comment]
Cannot reproduce it. Can you provide the full error stack please?

**Comment by 15556133244** [comment]
![图片](https://user-images.githubusercontent.com/46469612/60090271-ce05d500-9774-11e9-8c9f-8fae97b14f90.png)
![图片](https://user-images.githubusercontent.com/46469612/60090379-03122780-9775-11e9-98a2-531f87d5d5d6.png)

**Comment by 15556133244** [comment]
2.11.0
good at https://www.sis001.com
bad at https://www.sis001.com/forum/index.php

**Comment by tophf** [comment]
It'll be fixed when #630 is merged.
